charts/tidb-operator/values.yaml

```diff
@@ -1,9 +1,19 @@
 # Default values for tidb-operator
 
 # clusterScoped is whether tidb-operator should manage kubernetes cluster wide tidb clusters
-# Also see rbac.create and controllerManager.serviceAccount
+# Also see rbac.create, controllerManager.serviceAccount, scheduler.create and clusterPermissions.
 clusterScoped: true
 
+# clusterPermissions are some cluster scoped permissions that will be used even if `clusterScoped: false`.
+# NOTE: `clusterPermissions: {}` means `false` for all following permissions,
+# but not defined `clusterPermissions` (commented out) means `true` for all following permissions for back compatibility.
```

Comment:
I think we can have the same logic for these two cases, no need to differentiate them.

Reply:
both of them mean `true` for all?